portfolio/src/main/webapp/style.css

```diff
@@ -11,8 +11,8 @@ body {
 }
 
 #fullpage {
-  min-width: 100vw;
-  min-height: 100vh;
+min-width: 100vw;
```

Comment:
indent this and the next line by 2 spaces and swap them to maintain alphabetic order of declarations

Reply:
Updated with this commit 500fcf1